content/master/cli/command-reference.md

```diff
@@ -856,135 +851,182 @@ A Kubernetes cluster running Crossplane isn't required.
 {{< /table >}}
 
 #### Validate Common Expression Language rules
```

Comment:
Maybe this could be moved as the last header in the validate section? This is a complex validation case. The reason I had it at the top of the interactive workshop is to highlight that it was a recent addition to the validate functionality.